e2e_test.bash

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+log() {
+  echo -e "\n>>>>> $1\n"
```

Comment:
To make this easier to debug, use `set -ex` instead of `try_command` so it prints each of its commands and fails entirely on the first command failure

Reply:
Done